eg/Classes/PluginInstall.py

```diff
@@ -283,3 +431,154 @@ def VisitName(self, node):
 
     def VisitStr(self, node):
         return node.s
+
+
+class HiddenSystemDialog(wx.Dialog):
+
+    def __init__(self, fileFolders):
+
+        self.fileFolders = fileFolders
+        self.root = None
+
+        wx.Dialog.__init__(
+            self,
+            None,
+            title='Plugin Install - Hidden/System Files',
+            size=(450, 525),
+            style=(
+                wx.CAPTION |
+                wx.CLOSE_BOX |
+                wx.SYSTEM_MENU |
+                wx.RESIZE_BORDER |
+                wx.MAXIMIZE_BOX
+            )
+        )
+
+        headerBox = eg.HeaderBox(
+            self,
+            name='Plugin Install',
+            text=(
+                'There are hidden and system files please select\n'
+                'the files you wish to add to the egplugin package.'
```

Comment:
I have a question on this bit of it..

Do you think we should put a Text class into each bit we do. or should be put all core Text class items into eg.Text. because I have put everything into the core Text class so far. 

personally I would rather see a text class in each of the different files. for the various components. it makes it much easier when editing. because everything related to that specific piece is in one file. 

whatever you would like me to do.